### PR TITLE
HPCC-28529 Document Landing Zone Scopes

### PR DESCRIPTION
**Pull request by g-pan**
## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

## Testing:

Successfully executed unit tests: http://10.224.20.18/view/Docs-gp/job/Doc-Build-04-GPRepo2/

### CODE REVIEW

**Comment by g-pan** [comment]
why did this pull in Jakes split from 9.2.2 commit?? How can I separate that?

File: docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/UserSecurityMaint.xml [hunk 1]
```diff
@@ -1979,6 +1979,131 @@
     </sect3>
   </sect2>
 
+  <sect2 id="GS-LZ-Scopes">
+    <title>Landing Zone Security</title>
+
+    <para>You can set additional security options on Landing Zone(s). Feature
+    level security allows you to set permissions on access to your Landing
+    Zones and what users or groups can do there. Landing Zone Scope Security
+    allows you to set permissions on sub-folders in a Landing zone. This
```
Comment (JamesDeFabia) [inline docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/UserSecurityMaint.xml:1988]: consistency: Landing Zone

File: docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/UserSecurityMaint.xml [hunk 2]
```diff
@@ -1979,6 +1979,131 @@
...
+      accomplish this. .</para>
+    </sect3>
+
+    <sect3 id="LZ_FileScopes">
+      <title>Landing Zone File Scopes</title>
+
+      <para>File Scope Level Authorization provides a means to secure access
+      to folders within a Landing Zone.</para>
+
+      <para>An HPCC Administrator can define one or more Landing Zone scopes
```
Comment (JamesDeFabia) [inline docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/UserSecurityMaint.xml:2059]: Can a folder have more than one scope? "An HPCC Administrator can define one or more Landing Zone scopes for each folder..." implies that you can. But, later you say each folder is one scope and each scope is one folder.

File: docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/UserSecurityMaint.xml [hunk 3]
```diff
@@ -1979,6 +1979,131 @@
...
+      for each folder in an HPCC Landing Zone.</para>
+
+      <para>Each scope is a file folder of an HPCC Landing Zone. Each Landing
+      Zone scope is one HPCC file scope.</para>
+
+      <para>The landing zone file scopes can be defined using ECLWatch for
```
Comment (JamesDeFabia) [inline docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/UserSecurityMaint.xml:2065]: consistency:  Landing Zone

File: docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/UserSecurityMaint.xml [hunk 4]
```diff
@@ -1979,6 +1979,131 @@
...
+      security enabled systems.</para>
+
+      <para><graphic fileref="../../images/LDAP_0058-1.jpg"
+      vendor="ECLWatchSecurity" />To create a new file scope landing zone
+      folder, go to the Security page of ECL Watch, and click on
```
Comment (JamesDeFabia) [inline docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/UserSecurityMaint.xml:2070]: Awkward sentence: To create a new file scope landing zone folder. 
Perhaps: To create a new Landing Zone scope

File: docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/UserSecurityMaint.xml [hunk 5]
```diff
@@ -1979,6 +1979,131 @@
...
+      name and optionally a description.</para>
+
+      <para><graphic fileref="../../images/LDAP_0058-4.jpg"
+      vendor="ECLWatchSecurity" /></para>
+    </sect3>
+
+    <sect3 id="SEC_LZFilePermissions">
+      <title>Landing Zone File Permissions</title>
+
+      <para>You can set the landing zone file permissions according to your
```
Comment (JamesDeFabia) [inline docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/UserSecurityMaint.xml:2090]: consistency: Landing Zone

File: docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/UserSecurityMaint.xml [hunk 6]
```diff
@@ -1979,6 +1979,131 @@
...
+      requirements. Access your new landing zone using the following
```
Comment (JamesDeFabia) [inline docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/UserSecurityMaint.xml:2091]: consistency: Landing Zone

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
This PR may need rebasing (I see lots of files that I don't think you intended to include, such as YAML files). 
Additional comments inline

**Review by wangkx — COMMENTED** [review]
One comment

File: docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/UserSecurityMaint.xml (hunk 2 above)
Comment (JamesDeFabia) [reply docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/UserSecurityMaint.xml:2059]: This sentence is still confusing. The following sentence is a contradiction.

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
Comments inline

**Review by JamesDeFabia — COMMENTED** [review]
Looks ok now, but there are still many unrelated files included. Rebase, perhaps?

**Comment by g-pan** [comment]
Once @wangkx and @RussWhitehead (please review) approve content will look to rebase

**Review by wangkx — APPROVED** [review]
My comment has been addressed.

**Review by RussWhitehead — APPROVED** [review]
@g-pan Looks reasonable

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
The content for this Jira looks ok, but there are unrelated files included.

**Review by JamesDeFabia — APPROVED** [review]
Looks good from my POV now

**Pull request merged**